Content.Shared/Xenoarchaeology/Artifact/XAT/XATCompNearbySystem.cs

```diff
@@ -0,0 +1,26 @@
+using Content.Shared.Xenoarchaeology.Artifact.Components;
+using Content.Shared.Xenoarchaeology.Artifact.XAT.Components;
+
+namespace Content.Shared.Xenoarchaeology.Artifact.XAT;
+
+public sealed class XATCompNearbySystem : BaseXATSystem<XATCompNearbyComponent>
```

Comment:
need to check how is it used...

Reply:
ah, music instruments. Could be neat. But it looks so slow - to check on each update. Maybe better to add some limiters? like check every 2 seconds at least?...